src/main/java/org/genomenexus/vep_wrapper/VepController.java

```diff
@@ -98,4 +98,67 @@ public void fetchVepAnnotationByRegionsPOST(
         return;
     }
 
+    @RequestMapping(value = "/vep/human/hgvsc/{hgvsc}",
```

Comment:
Since database is optional, should we hide or disable this endpoint when no database?